optimum/habana/transformers/models/video_llava/processing_video_llava.py

```diff
@@ -0,0 +1,100 @@
+from typing import List, Optional, Union
+import numpy as np
+
+from transformers.models.video_llava.processing_video_llava import VideoLlavaProcessor
+from transformers.image_processing_utils import BatchFeature
+from transformers.image_utils import ImageInput, get_image_size, to_numpy_array
+from transformers.tokenization_utils_base import PaddingStrategy, PreTokenizedInput, TextInput, TruncationStrategy
+from transformers.utils import TensorType
+from optimum.utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+
+class GaudiVideoLlavaProcessor(VideoLlavaProcessor):
+    attributes = ["image_processor", "tokenizer"]
+    valid_kwargs = ["chat_template", "patch_size", "vision_feature_select_strategy", "image_token", "video_token"]
+    image_processor_class = "VideoLlavaImageProcessor"
+    tokenizer_class = "AutoTokenizer"
+
+    def __init__(
+        self,
+        image_processor=None,
+        tokenizer=None,
+        patch_size=None,
+        vision_feature_select_strategy=None,
+        image_token="<image>",  # set the default and let users change if they have peculiar special tokens in rare cases
+        video_token="<video>",
+        chat_template=None,
+        **kwargs,
+    ):
+        super().__init__(image_processor, tokenizer, chat_template=chat_template)
```

Comment:
@it sounds like you are missing to call the VideoLlavaProcessor constructor correctly?  and only calling ProcessorMixin constructor similar to the upstream code. This might cause issues